cursive-core/src/printer.rs

```diff
@@ -118,16 +118,47 @@ impl<'a, 'b> Printer<'a, 'b> {
         }
     }
 
+    /// Prints some text at the given position.
+    ///
+    /// # Parameters
+    /// * `start` is the offset used to print the text in the view.
+    /// * `text` is a simple string to print on the screen.
```

Comment:
We can also specify that `text` should really be a single line, no line-wrapping will be done here.